migration-engine/connectors/sql-migration-connector/src/sql_renderer.rs

```diff
@@ -49,15 +49,12 @@ pub(crate) trait SqlRenderer {
             .columns
             .iter()
             .map(|column| {
-                // FIXME Temporary hack: we should get this from a `TableRef`, but
-                // this is not possible because we sometimes create tables as
-                // part of the table redifinition process on sqlite.
-                let column_ref = ColumnRef {
-                    column,
```

Comment:
I think the FIXME still applies, because we are constructing artificial `ColumnRef`s on tables that do not exist in the target schema. Other than that :+1: